pynxtools/dataconverter/readers/stm/stm_helper.py

```diff
@@ -0,0 +1,224 @@
+"""
+    Some generic function and class for on STM reader.
+"""
+# Copyright The NOMAD Authors.
+#
+# This file is part of NOMAD. See https://nomad-lab.eu for further info.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Tuple
+import json
+import numpy as np
+from pynxtools.dataconverter.helpers import convert_data_dict_path_to_hdf5_path
+
+
+# Here are some data or data type or unit or data to skip:
+UNIT_TO_SKIP = ['on/off', 'off', 'on', 'off/on']
```

Comment:
map to boolean instead, I have seen such example also from mpes, could be moved into a helper for the converter to be shared across all pynxtools readers